.github/workflows/e2e.yml

```
@@ -39,7 +41,7 @@ jobs:
         run: npx playwright install chromium --with-deps
 
       - name: Run dev server
-        run: yarn start --sources 'packages/esm-*-app/'&
+        run: yarn start --sources 'packages/esm-*-app/' --port 8180 &
```

```suggestion
        run: yarn start --sources 'packages/esm-*-app/' --port 8180 & # Refer to O3-1994
```

It would be better to keep a comment mentioning the reason behind this change. 

cc: @jayasanka-sack